source/agora/test/Base.d

```diff
@@ -1683,8 +1775,7 @@ public APIManager makeTestNetwork (APIManager : TestAPIManager = TestAPIManager)
         conf.node.genesis_start_time = genesis_start_time;
     }
 
-    immutable(Block)[] blocks = generateBlocks(gen_block,
-        test_conf.extra_blocks);
+    immutable(Block)[] blocks = [gen_block];
```

Comment:
I think in a future PR we can remove this too. But you're already planning this right?

Reply:
Correct. Will be next PR.